frontend/src/styles/global.scss

```diff
@@ -681,6 +681,11 @@ body {
         }
     }
 
+    .ant-input-number:hover,
+    .ant-input-number-focused {
+        border-color: var(--primary-3000);
```

Comment:
Not sure we actually want this to be the primary color.

Since https://github.com/PostHog/posthog/pull/18746 we use `--border-bold` for the input active states
https://github.com/PostHog/posthog/blob/1d85df34e09573fd41af9afd955691c98f14b846/frontend/src/lib/lemon-ui/LemonInput/LemonInput.scss#L18-L32

Reply:
Ah fun, I've styled them from blue -> orange. But `--border-bold` actually makes a lot more sense now.